## Bem vindo!

Welcome to our lab! We are part of the [MostMicro](http://www.itqb.unl.pt/mostmicro) research unit at [ITQB NOVA](http://www.itqb.unl.pt/), an institute of [Universidade NOVA de Lisboa](http://www.unl.pt/). The lab started in January 2017.

## Mission Statement

Our lab answers questions in microbial molecular biology, with a focus on single-molecule fluorescence microscopy in living cells. By detecting, tracking and counting individual DNA, mRNA and protein molecules, we will learn how to make better synthetic genetic circuits, how gene expression is regulated, and how cells interact with each other, other organisms, and their environments. We are especially interested in how single-cells evolve over time: our experiments complement modern high-resolution and high-throughput approaches that look at single time points. We closely follow developments in microscopy (especially emerging 3D imaging techniques) and fluorescent labeling. The lab embraces the open science revolution, with strong preferences for open access publication, data/code availability, and preprint.

## Current work

Check out our most recent work on [a protein complex essential for remodelling the *E. coli* cell wall](https://www.nature.com/articles/s41467-023-39921-4) every cell time the cell divides. In collaboration with [Jie Xiao's lab](https://xiao.med.jhmi.edu/) and [Albert Lau's lab](https://lau.med.jhmi.edu/), we investigated the structure and dynamics of this "divisome" to examine how variations at key protein-protein interfaces could impact essential enzymatic activities. Now, we are applying insights from this method to look at how cell division can respond to environmental changes.
![molecular dynamics of the divisome complex](/img/divisome.gif){: .blog-image-center-noresize }*Dynamics of the bacterial divisome*{: .blog-image-caption }

Also see our [low-noise gene expression system](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0187259) in *E. coli* ([plasmids available at AddGene](https://www.addgene.org/Zach_Hensel/)). We developed this to facilitate single-molecule experiments with fluorescent proteins that require very low expression levels with very low cell-to-cell variation. In João Silva's thesis work, we subsequently extended this for [independent tuning of low-noise expression of two proteins from two plasmids](https://journals.asm.org/doi/full/10.1128/msphere.00340-19).

![low-noise expression from 100 to 10,000 molecules/cell](/img/lownoise.jpg){: .blog-image-center }*Gene expression from 100 to 10,000 molecules/cell with low cell-to-cell variation*{: .blog-image-caption }  

We are using this system to develop and improve methods for live-cell mRNA imaging. Browse the [research](research.md) page for details about our other projects.

{:refdef: style="text-align: center;"}
![live-cell mRNA imaging in E. coli](/img/timelapseRNA.gif){: .blog-image-center-noresize }*Imaging single mRNAs in E. coli*{: .blog-image-caption }
{: refdef}

## Funding

Our work is financially supported by the Fundação para a Ciência e a Tecnologia (FCT) through MOSTMICRO-ITQB (UIDB/04612/2020, UIDP/04612/2020) and LS4FUTURE (LA/P/0087/2020).
